stdlib/builtins.pyi

```diff
@@ -53,7 +53,7 @@ from typing import (  # noqa: Y022
     overload,
     type_check_only,
 )
-from typing_extensions import (
+from typing_extensions import (  # type: ignore
```

Comment:
These type ignores look like some pretty weird mypy bugginess surrounding LiteralString. I thought probably makes sense to just ignore for now and figure it out later, so we can test 3.12 effectively

Reply:
Were you using mypy `master` when you ran tests locally for this PR? This looks like the regression I reported in https://github.com/python/mypy/pull/15161#issuecomment-1531596612

Reply:
Ah I'd missed that, it's definitely the same thing. Hmm, I'm not sure what the best way to proceed is.